test/jdk/java/util/zip/ZipFile/TestZipFileEncodings.java

```diff
@@ -247,7 +304,7 @@ private void writeEntry(ZipOutputStream zos, CRC32 crc,
                 crc.update(data);
                 ze.setCrc(crc.getValue());
             }
-            ze.setTime(System.currentTimeMillis());
+            ze.setTime(1675862371399L);
```

Comment:
Please add a comment indicating what the time is

Reply:
This change was accidentaly introduced, I have reverted it. Good catch.